services.yml

```diff
@@ -50,6 +50,11 @@ services:
   Webgriffe\AmpElasticsearch\Client:
     arguments:
       $baseUri: '%elasticsearch_base_uri%'
+    calls:
+      - method: 'setCredentials'
+        arguments:
+          - '%elasticsearch_user%'
+          - '%elasticsearch_password%'
```

Comment:
What if someone is working with Elasticsearch without the security pack, meaning no user/password authentication is required? We need to ensure backward compatibility and support scenarios where a username and password are not provided.